addons/point_of_sale/static/tests/tours/point_of_sale.js

```diff
@@ -1,3 +1,4 @@
+/* global posmodel */
```

Comment:
really? the point of sale has a global posmodel object?

Reply:
Oh yes: https://github.com/odoo/odoo/blob/54922b38ec2a7de6c3691bae1b23b7288236d04b/addons/point_of_sale/static/src/js/models.js#L72